packages/client/hmi-client/src/page/Home.vue

```diff
@@ -33,56 +33,179 @@
 									class="p-inputtext-sm"
 								/>
 							</span>
+							<MultiSelect
+								v-if="view === ProjectsView.Table"
+								:modelValue="selectedColumns"
+								:options="columns"
+								:maxSelectedLabels="1"
+								optionLabel="header"
+								@update:modelValue="onToggle"
+								placeholder="Add or remove columns"
+								class="p-inputtext-sm"
+							/>
+							<span class="p-buttonset">
+								<Button
+									class="p-button-secondary p-button-sm"
+									label="Cards"
+									icon="pi pi-image"
+									@click="view = ProjectsView.Cards"
+									:active="view === ProjectsView.Cards"
+								/>
+								<Button
+									class="p-button-secondary p-button-sm"
+									label="Table"
+									icon="pi pi-list"
+									@click="view = ProjectsView.Table"
+									:active="view === ProjectsView.Table"
+								/>
+							</span>
 						</section>
-						<div v-if="!isLoadingProjects && isEmpty(tab.projects)" class="no-projects">
-							<img src="@assets/svg/seed.svg" alt="" />
-							<template v-if="tab.title === 'My projects'">
-								<h3>Welcome to Terarium</h3>
-								<div>
-									Get started by creating a
-									<Button
-										label="new project"
-										class="p-button-text new-project-button"
-										@click="isNewProjectModalVisible = true"
-									/>. Your projects will be displayed on this page.
-								</div>
-							</template>
-							<template v-else-if="tab.title === 'Shared projects'">
-								<h3>You don't have any shared projects</h3>
-								<p>Shared projects will be displayed on this page</p>
-							</template>
-						</div>
-						<div v-else class="carousel">
-							<div class="chevron-left" @click="scroll('left', $event)">
-								<i class="pi pi-chevron-left" />
+						<section class="list-of-projects">
+							<div v-if="!isLoadingProjects && isEmpty(tab.projects)" class="no-projects">
+								<img src="@assets/svg/seed.svg" alt="" />
+								<template v-if="tab.title === 'My projects'">
```

Comment:
perhaps making the tab title an enum would be good here? i.e.
```
enum TabTitles = {
  MY_PROJECTS = 'My projects',
  SHARED_PROJECTS = 'Shared Projects'
}
```
Might be better for the future if statements or if the text were to change